org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ZipActionImpl.java

```diff
@@ -418,6 +433,25 @@ private void applyZipStream(
 		}
 	}
 
+	private void printZipActionDuration(final String inputName) {
+		if (getLogger().isInfoEnabled()) {
+			getLogger().info("Stop processing [ {} ] using [ {} ] took [ {}ms ]: {}",
+							 inputName, getName(),
+							 getActiveChanges().getElapsedMillis(), getActiveChanges().getChangeText());
+		}
+	}
+
+	private void printAdviseOnUnchanged(final String inputName) {
+		if (getLogger().isInfoEnabled() && !getActiveChanges().isChanged()) {
```

Comment:
What is the value of this? I think we already report the  number of changes in an element which may be zero thus providing this information.

Also, the advice is rather specific to a specific artifact when transformation rules should be more general to the domain. Updating the artifact to a later version without removing the advised exclusions can result in a partial transformation of the artifact if the excluded element in the newer artifact does indeed require transformation.

Reply:
This is a valid point. Of course if you change the rules, you should always run without any exclusions. But if you do iterations, or if you start converting some of your applications dependencies to jakarta, you can safely exclude them from the process.